pkg/rehearse/jobs.go

```diff
@@ -496,3 +603,15 @@ func (e *Executor) submitRehearsal(job *prowconfig.Presubmit) (*pjapi.ProwJob, e
 
 	return e.pjclient.Create(&prowJob)
 }
+
+func (e *Executor) submitPeriodic(job prowconfig.Periodic) (*pjapi.ProwJob, error) {
+	labels := make(map[string]string)
+	for k, v := range job.Labels {
+		labels[k] = v
+	}
+
+	prowJob := pjutil.NewProwJob(pjutil.PeriodicSpec(job), labels)
+	e.loggers.Job.WithFields(pjutil.ProwJobFields(&prowJob)).Info("Submitting a new prowjob.")
+
+	return e.pjclient.Create(&prowJob)
+}
```

Comment:
Only `job` differs between these two functions, can they be DRYed?

Reply:
nope. the job is different type. I could use some reflection but I don't think its a good idea.

Reply:
The functions are now so small that the duplication is fine.